samples/word/10-content-controls/insert-and-change-checkbox-content-control.yaml

```diff
@@ -0,0 +1,162 @@
+order: 8
+id: word-content-controls-insert-and-change-checkbox-content-control
+name: Manage checkbox content controls
+description: 'Inserts, updates, and retrieves checkbox content controls.'
+host: WORD
+api_set:
+    WordApi: '1.7'
+script:
+    content: |
+        $("#insert-controls").on("click", () => tryCatch(insertCheckboxContentControls));
+        $("#change-controls").on("click", () => tryCatch(toggleCheckboxContentControls));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function insertCheckboxContentControls() {
+          // Traverses each paragraph of the document and places a checkbox content control at the beginning of each.
+          await Word.run(async (context) => {
+            let paragraphs = context.document.body.paragraphs;
+            paragraphs.load("$none"); // Don't need any properties; just start each paragraph with a content control.
+
+            await context.sync();
+
+            for (let i = 0; i < paragraphs.items.length; i++) {
+              let contentControl = paragraphs.items[i]
+                .getRange(Word.RangeLocation.start)
+                .insertContentControl(Word.ContentControlType.checkBox);
+            }
+            console.log("Checkbox content controls inserted: " + paragraphs.items.length);
+
+            await context.sync();
+          });
+        }
+
+        async function toggleCheckboxContentControls() {
```

Comment:
Not critical, but what do you think of adding another button to sample check/uncheck the checkbox where the cursor is?  That way, you can show different ways to access a content control.

Reply:
Added - see toggleCheckboxContentControl